pages/docs/recipes/store-values.en-US.mdx

```diff
@@ -86,6 +86,38 @@ userStore.set((state) => state.user = /* value from somewhere */);
 
 Updating the store using `set()` will update all variables subscribed to the store, including (like in the example) `user`.
 
+#### App context
+
+Provided by `@openmrs/esm-framework` it`s another way to store data in O3. If you are familiar with [React's context](https://react.dev/reference/react/useContext#reference) it works similarly in the same way. All you
+need is to create a global context in some higher level component and access it somewhere in any of its children.
+The `useDefineAppContext` hook takes in the name of your context and its initial values.
```

Comment:
```suggestion
Provided by `@openmrs/esm-framework` it's another way to store data in O3. If you are familiar with [React's context](https://react.dev/reference/react/useContext#reference), it works similarly. Unlike React's Context, App Context works
across React subtrees, and thus can work in extensions and workspaces. All you need is to create a global context in
some higher level component and access it somewhere in any of its children. The `useDefineAppContext` hook takes in
the name of your context and its initial values.
```

Reply:
Thanks @chibongho I like the suggestion 👍 .